lib/ohloh_scm/adapters/git/pull.rb

```diff
@@ -104,6 +105,8 @@ def convert(source_scm)
 					commit_all(r)
 				end
 				yield(commits.size, commits.size) if block_given?
+			elsif !read_token && commits.empty?
+				raise RuntimeError, "Empty repository"
```

Comment:
Do we have something that will handle the `RuntimeError` so that the jobs do not continue to crash under this circumstance?